Sources/TuistCore/Graph/GraphDependencyReference.swift

```diff
@@ -26,7 +26,7 @@ public enum GraphDependencyReference: Equatable, Comparable, Hashable {
         product: Product
     )
     case bundle(path: AbsolutePath)
-    case product(target: String, productName: String, platformFilter: BuildFilePlatformFilter? = nil)
+    case product(target: String, productName: String, platformFilter: PlatformFilter? = nil)
```

Comment:
Isn't it possible to have multiple platform filters?

Reply:
Probably.  Here i just updated the type but did not assess any logic changes within the Graph related code.  If this is a path we want to take then this will probably need to be an array or Set